changelog/58938.added

```diff
@@ -0,0 +1,24 @@
+Salt's versions report `salt --versions-report` now includes all installed salt extensions into its versions report.
+
+```
+Salt Version:
```

Comment:
**suggestion**: I don't think we want an example here. Changeling entries should be 1 line. They will link back to the issue/pr for more context

Reply:
Resolved